README.md

```diff
@@ -0,0 +1,54 @@
+# at2-node
+
+[![at2-node](https://github.com/Distributed-EPFL/at2-node/actions/workflows/rust.yml/badge.svg)](https://github.com/Distributed-EPFL/at2-node/actions/workflows/rust.yml)
+[![codecov](https://codecov.io/gh/Distributed-EPFL/at2-node/branch/master/graph/badge.svg)](https://codecov.io/gh/Distributed-EPFL/at2-node)
+
+An implementation of a distributed ledger using
+[AT2](https://arxiv.org/abs/1812.10844) (Asynchronous Trustworthy Transfers).
+It allows to send money to a chosen identity provided that you have enough in
```

Comment:
```suggestion
It allows to send tokens to a chosen identity provided that you have enough in
```

Reply:
Also allow is not the proper verb here, a better phrasing would be something along the lines of `It provides an easy way to transfer tokens from identity to identity provided there are enough in the source identity's account`

Reply:
Hum, I'm using "money" throughout the code, even in the CLI. Keeping it as it makes it clearer IMO

Reply:
Don't use `money`. It should be either one of `token`, `asset`, or `coin`. Even if I think the latter is also too much noted into fiat money. Following the name of the project, it should probably be `asset`. `Token` is probably better for the NonFungibleTokens...

Reply:
Alright, "asset" it is!